content/desktop/feedback.md

```diff
@@ -34,10 +34,10 @@ GitHub](https://github.com/docker/desktop-linux/issues)
 
 #### Feedback via Community Slack channels
 
-You can also provide feedback through the following [Docker Community Slack](https://www.docker.com/community/) channels:
+You can also provide feedback through the following [Docker Community Slack](https://dockr.ly/slack) channels:
```

Comment:
I think this one is broken, as it doesn't redirect to a valid invite-URL (so only can be used for people with an `@docker.com` e-mail address.